src/main/java/net/mcreator/element/types/interfaces/IAnchorableElement.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.types.interfaces;
+
+import net.mcreator.ui.init.L10N;
+
+public interface IAnchorableElement {
+
+	AnchorPoint getAnchorPoint();
+
+	enum AnchorPoint {
+		TOP_LEFT, TOP_RIGHT, CENTER, BOTTOM_LEFT, BOTTOM_RIGHT;

```

Comment:
I thought about it earlier today, but I think good other anchor points would be things like `CENTER_TOP`, `CENTER_BOTTOM`, `CENTER_RIGHT` and `CENTER_LEFT`. I'm quite sure they will be requested in the future.

Reply:
Good point,